events/job_offmap.txt

```diff
@@ -973,17 +973,20 @@ character_event = {
 	desc =  MNM.70011.d
 	picture = GFX_evt_melee
 	border = GFX_event_normal_frame_war
-	
+
 	is_triggered_only = yes
-	
+
 	immediate = {
 		if = {
 			limit = {
 				event_target:commander_improved = {
 					NOR = {
 						trait = light_foot_leader
-						trait = heavy_infantry_leader 
```

Comment:
Put heavy_infantry_leader to its original place so that the GitHub will stop thinking that you removed it.